### PR TITLE
Fix for Issue #165. Documentation (README) improvements

### PR DESCRIPTION
**Pull request by jfdenise**

### CODE REVIEW

**Comment by jfdenise** [comment]
@bstansberry some doc improvment #168

File: README.md [hunk 1]
```diff
@@ -14,18 +28,95 @@ Galleon feature-pack compatible with WildFly Preview
 
 The Maven coordinates to use is: `org.wildfly:wildfly-datasources-preview-galleon-pack:<version>`
 
-Datasources
-=======
+Supported database types
+===============
+
+For each database type its support, each feature-pack provides 3 Galleon layers that build upon each other :
```
Comment (bstansberry) [inline README.md:34]: s/its support/it supports/g

File: README.md [hunk 2]
```diff
@@ -14,18 +28,95 @@ Galleon feature-pack compatible with WildFly Preview
...
+
+* One just to add the driver (eg: `postgresql-driver`).
```
Comment (bstansberry) [inline README.md:36]: Perhaps add:

This installs a JBoss Modules module for the driver and adds a driver resource to the datasources subsystem in the server configuration.

File: README.md [hunk 3]
```diff
@@ -14,18 +28,95 @@ Galleon feature-pack compatible with WildFly Preview
...
-* [MariaDB](doc/mariadb/README.md)
+```
+galleon.sh provision provisioning.xml --dir=my-wildfly-server
+```
 
-* [Microsoft SQL Server](doc/mssqlserver/README.md)
+## Provisioning using the [WildFly Maven Plugin](https://github.com/wildfly/wildfly-maven-plugin/) or the [WildFly JAR Maven plugin](https://github.com/wildfly-extras/wildfly-jar-maven-plugin/)
 
-* [MySQL](doc/mysql/README.md)
+You need to include the datasources feature-pack and layers in the Maven Plugin configuration. This look like:
```
Comment (bstansberry) [inline README.md:98]: s/This look like/This looks like/g

or 

s/This look like/For example/g

**Review by bstansberry — CHANGES_REQUESTED** [review]
Very nice @jfdenise! And thanks. I had this issue on my personal todo list, but am unlikely to get to it any time soon, so much appreciated!

I've made various copy editing comments but it looks great. Please go ahead and merge when you are ready.

**Comment by jfdenise** [comment]
@bstansberry thank-you for the review. Merged with your feedback.

**Review by bstansberry — COMMENTED** [review]
Thanks, Jean-Francois. :)

**Pull request merged**